libevent-sys/build.rs

```diff
@@ -30,6 +30,18 @@ fn build_libevent(libevent_path: impl AsRef<std::path::Path>) -> PathBuf {
         }
 
         config.register_dep("openssl");
+
+        let openssl_root = if let Ok(root) = env::var("DEP_OPENSSL_ROOT") {
```

Comment:
For posterity, this metadata was only added in openssl-sys 0.9.48 (https://github.com/sfackler/rust-openssl/commit/ef86438a109f7f7683bde078563750a3ed02994d), and the logic here exists to be backwards-compatible with earlier versions.